cloudify_agent/shell/commands/daemons.py

```diff
@@ -161,6 +157,11 @@ def create(**params):
     custom_arg = attributes.pop('custom_options', ())
     attributes.update(_parse_custom_options(custom_arg))
     click.echo('Creating...')
+    manager_ip = get_manager_ip(attributes)
```

Comment:
Well, I need 4 different attributes from there, and I also set some of the attributes back into the dict, so it's easiest for me to just have the whole dict there